docs-src/concepts/what-is-visibility.md

```diff
@@ -7,4 +7,10 @@ tags:
   - term
 ---
 
-The term Visibility, within the Temporal Platform, refers to the subsystems and APIs that enable an operator to view Workflow Executions that currently exist within a Cluster.
+The term [Visibility](/visibility), within the Temporal Platform, refers to the subsystems and APIs that enable an operator to view Workflow Executions that currently exist within a Cluster.
+
+The [Visibility store](/cluster-deployment-guide#visibility-store) in your Temporal Cluster stores the Workflow Execution Event History data and is set up as a part of your [Persistence store](/concepts/what-is-a-temporal-cluster#persistence) to enable listing and filtering details about Workflow Executions that exist on your Temporal Cluster.
```

Comment:
> stores the Workflow Execution Event History data

Could this confuse people and make people think visibility store also contains workflow history?